server/requirements.txt

```diff
@@ -40,6 +40,6 @@ python3-ldap==0.9.5.3
 behave==1.2.5
 wooper==0.4.2
 
-git+git://github.com/superdesk/eve@sync-upstream#egg=Eve==0.6-dev
```

Comment:
smart is always better, we need to make smart things also :)
Thanks for help Peter.